.github/workflows/publish-master-merges.yaml

```diff
@@ -0,0 +1,56 @@
+name: Publish all merges to master
+
+on:
+  push:
+    branches: ["publish_chart_on_merge"]
+    paths:
+      - helm-chart/**
+      - scripts/**
+      - acceptance-tests/**
+      - .github/**
+  workflow_dispatch:
+
+jobs:
+  publish-chart:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions-ecosystem/action-get-latest-tag@v1
+        id: get-latest-tag
+      - uses: actions-ecosystem/action-bump-semver@v1
+        id: bump-semver
+        with:
+          current_version: ${{ steps.get-latest-tag.outputs.tag }}
+          level: prepatch
+      - id: set-version
+        run: echo "publish_version=${{ steps.bump-semver.outputs.new_version }}.$(echo ${{ github.sha }} | cut -c 1-7)" >> $GITHUB_ENV
+      - uses: SwissDataScienceCenter/renku-actions/publish-chart@v0.3.0
+        env:
+          CHART_DIR: helm-chart/
+          CHART_TAG: "--tag ${{env.publish_version}}"
+          CHART_NAME: renku
+          GIT_USER: renku-bot
+          GIT_EMAIL: renku@datascience.ch
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+          GITHUB_TOKEN: ${{ secrets.RENKUBOT_GITHUB_TOKEN }}
+      - uses: 8398a7/action-slack@v3
+        with:
+          status: custom
+          fields: job,ref
+          custom_payload: |
+            {
+              attachments: [{
+                color: '${{ job.status }}' === 'success' ? 'good' : '${{ job.status }}' === 'failure' ? 'danger' : 'warning',
+                text: `${process.env.AS_JOB} version ${{ env.publish_version }}: ${{ job.status }}.`,
+              }]
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        if: always()
+      - uses: actions-ecosystem/action-push-tag@v1
+        with:
+          tag: ${{ steps.bump-semver.outputs.new_version }}
```

Comment:
this tag is wrong (see [here](https://github.com/SwissDataScienceCenter/renku/runs/4468084248?check_suite_focus=true#step:11:9)) - should be `${{env.publish_version))`?

Reply:
oh, damn, yes, thanks!